packages/odyssey-storybook/src/components/odyssey-labs/DataView/DataView.stories.tsx

```diff
@@ -526,6 +531,7 @@ export const ExpandableRowsAndCards: Story = {
         tableLayoutOptions={{
           columns: personColumns,
           renderDetailPanel: renderAdditionalContent,
+          expandColumnPosition: 0,
```

Comment:
Instead of modifying the existing story, do we have a separate test available for moving this expand column?

Also, wouldn't we only want it at left or right and not in the middle of the row?

Instead of an index, I'm thinking this would be better suited to a string identifier union `"left" | "right"` or `"start" | "end"` because we need it to work with right-to-left modes.